src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,43 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
 public class IndexController extends HttpServlet {
```

Comment:
Note:
You can use for initializing servlet annotation.
So : @WebServlet(urlPatterns = "/index")
If you do it, you don't need a lot of code in web.xml.
It was in the videos.

Reply:
I know, but I didn't refactor, because it is not myself was written.